### ISSUE TITLE
The Celebration ToK entry is not updating

### ISSUE THREAD

**Issue by Skullgrin** [WarEmu/WarBugs#17127]
Finished the final quest to kill the Snot, and the 'Madness Reigns" quest completed, however the ToK quest for kill the Snot did not complete, and the quest count did not update either (it is stuck at 8/10 now).

![Thargrimm_559](https://user-images.githubusercontent.com/40448982/84599331-7ef5fb80-ae2e-11ea-9016-62d1a12a6357.jpg)

I really hope there is a way for someone to fix this, I would really like to have the cape...

**Comment by Skullgrin** [comment]
Update to the original post: I went back and did "kill the Snot" again, and it gave credit for the Quest and advanced the count to 9/10 completed.

![Thargrimm_574](https://user-images.githubusercontent.com/40448982/84686651-9948db80-aef9-11ea-988a-1bd5ad22eda2.jpg)

After restarting the client the event counter finally updated to 10/10, so I was able to finally complete the event. Hope this helps in some way to determine just what went wrong initially.
